packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -108,8 +108,9 @@ export class Tooltip {
   @Method()
   async showTooltip(anchorElement: any) {
     clearTimeout(this.hideTooltipTimeout);
-    this.visible = true;
-    this.computeTooltipPosition(anchorElement);
+    this.computeTooltipPosition(anchorElement).then(() => {
+      this.visible = true;
+    });
```

Comment:
```suggestion
    await this.computeTooltipPosition(anchorElement);
    this.visible = true;
```

Please use await/async to handle promises.

Reply:
done